Example/BonMot/BONSpecialGenerator.swift

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env swift
```

Comment:
I think I fixed that just now.  I deleted the version under Examples, and referenced the one in ./scripts.